### ISSUE TITLE
[Bug]: v9 Spinner: Primary with label: Elements must only use allowed ARIA attributes

### ISSUE THREAD

**Issue by jurokapsiar** [microsoft/fluentui#22662]
### Library

React Components / v9 (@fluentui/react-components)

### System Info

```shell
vr-tests storybook
```

### Are you reporting Accessibility issue?

yes

### Reproduction

vr-tests storybook

## Actual Behavior
Primary with label: Elements must only use allowed ARIA attributes (ARIA attribute cannot be used, add a role attribute or use a different element: aria-labelledby) 

## Expected Behavior
he label needs to move to the element with role progressbar

### Requested priority

High

### Are you willing to submit a PR to fix?

no

### Validations

- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] The provided reproduction is a minimal reproducible example of the bug.
